server/dash/dashboards/neighborhood.py

```diff
@@ -18,12 +18,15 @@
 # TITLE
 title = "NEIGHBORHOODS"
 
+
 # DATA
 print(" * Downloading data for dataframe")
 query_string = '/reports?field=type_name&field=council_name&field=created_date'
 df = pd.read_json(API_HOST + query_string)
 print(" * Dataframe has been loaded")
 
+df["created_date"] = pd.to_datetime(df['created_date'])
```

Comment:
Could you leave a comment explaining why we need to do this?

Reply:
This is no longer needed.

Reply:
got rid of it